src/plugins/localStorage/LocalStorageObjectProvider.js

```diff
@@ -83,7 +85,7 @@ export default class LocalStorageObjectProvider {
    * @private
    */
   getSpaceAsObject() {
-    return JSON.parse(this.getSpace());
+    return JSON.parse(this.getSpace(), filter__proto__);
```

Comment:
Can we use Vue's `toRaw`utility here?
Something like:
`return JSON.parse(toRaw(this.getSpace())`